clc/modules/block-storage/src/main/java/com/eucalyptus/blockstorage/DASManager.java

```diff
@@ -787,6 +787,46 @@ public boolean shouldExportOnReload(LVMVolumeInfo lvmVolumeInfo) {
       return returnValues;
     }
 
+    protected String enableLogicalVolume(String lvName) throws EucalyptusCloudException {
+      
+      EucalyptusCloudException ex = null;
+      String result = null;
+
+      // 1st timeout is 10 ms, 2nd is 40. Last (9th) is 2.56 sec. Total 3.41 sec.
+      int attempt = 1;
+      final int max_attempts = 9;
+      int timeout = 10; //ms
+      final int timeout_multiplier = 4;
+
+      do {
+        try {
+          result = LVMWrapper.enableLogicalVolume(lvName);
+          if (attempt > 1) {
+            LOG.info("Enabling " + lvName + "succeeded on retry.");
```

Comment:
Avoid conditional logging. If you'd like to note the attempt in the logs, log the action and the attempt prior to invoking LVMWrapper function

`LOG.debug("Enabling logical volume " + lvName + ", attempt " + attempt);`